### PR TITLE
editorconfig update

### PR DESCRIPTION
**Pull request by RebeccaStevens**
Extracted out of #215

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -3,5 +3,17 @@ root = true
...
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.yml]
+indent_style = space
+indent_size = 2
+
+[{package.json,package-lock.json}]
+indent_style = space
```
Comment (TehShrike) [inline .editorconfig:18]: I recently realized that at some point the npm cli was updated to not clobber your chosen indentation in package.json on every save, so if I'm bothering to specify it explicitly, I'd be a fan of tabs

File: .editorconfig [hunk 2]
```diff
@@ -3,5 +3,17 @@ root = true
 [*]
 indent_style = tab
 end_of_line = lf
+charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (TehShrike) [inline .editorconfig:11]: I don't use/like the "newlines at end of lines add manual linebreaks" feature of markdown, so I would eliminate this

Comment (RebeccaStevens) [reply .editorconfig]: fixed

File: .editorconfig (hunk 1 above)
Comment (RebeccaStevens) [reply .editorconfig]: I removed this and updated the package.json file to use tabs.

**Pull request merged**